controllers/routemonitor/routemonitor_controller.go

```diff
@@ -34,38 +34,47 @@ type RouteMonitorReconciler struct {
 	Scheme *runtime.Scheme
 }
 
-// +kubebuilder:rbac:groups=*,resources=services,verbs=get;list;watch;create;
-// +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;
-// +kubebuilder:rbac:groups=monitoring.coreos.com,resources=servicemonitors,verbs=get;list;watch;create;
+// +kubebuilder:rbac:groups=*,resources=services,verbs=get;list;watch;create
+// +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create
+// +kubebuilder:rbac:groups=monitoring.coreos.com,resources=servicemonitors,verbs=get;list;watch;create;delete
 // +kubebuilder:rbac:groups=monitoring.openshift.io,resources=routemonitors,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=monitoring.openshift.io,resources=routemonitors/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=route.openshift.io,resources=routes,verbs=get;list;watch
 
 func (r *RouteMonitorReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	ctx := context.Background()
+	log := r.Log.WithName("Reconcile")
 
-	routeMonitor, err := r.GetRouteMonitor(ctx, req)
+	log.Info("Entering GetRouteMonitor")
```

Comment:
Maybe `log.Debug` 
not sure we have to constantly log that on info level

Reply:
I can do that, but it will log it anyways, just with a DEBUG header